examples/README.md

```diff
@@ -46,7 +46,7 @@ $ ./infrastructure/mesh.sh
 $ ./infrastructure/ecs-cluster.sh
 ```
 
-* Setup EKS Cluster (Optional if using EKS). Note that there are more steps to use Kubernetes cluster that are not covered here. Please follow [EKS Getting Started Guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html).
+* Setup EKS Cluster (Optional if using EKS). Note that there are more steps to use Kubernetes cluster that are not covered here. Please follow [EKS Getting Started Guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html). See also the [Walkthrough: App Mesh with EKS](../walkthroughs/eks/) for others options.
```

Comment:
nit:others -> other

Reply:
Ha, eagle eye! Thanks and fixed in 533dbe3f34b1e16c52107c083384cd0786971159